### DISCUSSION TITLE
Should I make an overlay package for ULauncher for Gentoo?

### DISCUSSION THREAD

**Discussion by floweringnights** [Ulauncher/Ulauncher#1070] [Packaging]
I notice that there are 0 ways to install it on Gentoo, or things like NixOS and Void, so I want to host the packages on overlays for those distros

**Comment by friday** [comment]
We would be happy to accept contributions. Third party or adding to our scripts, so we can support them.

Packaging is a catch 22 where you have to be an expert in distros you haven't even used. Many formats are stupidly strict and hard to adhere to.

https://repology.org/project/ulauncher/versions lists some third party packages, including distros you mentioned. Some are rather outdated though.
